### PR TITLE
Add compat data for CSS [repeating-]radial-gradient

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This is to check off yet another listing on #2684.  All of the data is based off of the existing compatibility table, as well as a little backup from the data on https://caniuse.com.

### CODE REVIEW

**Comment by ExE-Boss** [comment]
Last I checked, gradients were being handled by @teoli2003 (#2792 and #2846)

**Comment by vinyldarkscratch** [comment]
Ah, were they also planning on tackling the `radial-gradient()` and `repeating-radial-gradient()`?

**Comment by vinyldarkscratch** [comment]
I assumed that they were taking care of just the `linear-gradient` and `repeating-linear-gradient`, and didn't see anything regarding `radial-gradient` or `repeating-radial-gradient`.  So, since it was something on the checklist of things that needed to be converted, I decided to try and tackle it, haha.  😛

**Comment by teoli2003** [comment]
I'll happily review this once PR #2846 is merged 👍

**Comment by vinyldarkscratch** [comment]
Thanks, @teoli2003!

File: css/types/image.json [hunk 1]
```diff
@@ -405,6 +405,533 @@
                 }
               }
             }
+          },
+          "radial-gradient": {
+            "__compat": {
+              "description": "<code>radial-gradient()</code>",
```
Comment (jpmedley) [inline css/types/image.json]: The `"mdn_url"` element is missing.

File: css/types/image.json [hunk 2]
```diff
@@ -405,6 +405,533 @@
...
+            },
+            "at": {
+              "__compat": {
+                "description": "<code>at</code> syntax",
+                "support": {
+                  "webview_android": {
+                    "version_added": true
+                  },
+                  "chrome": {
+                    "version_added": "26"
```
Comment (jpmedley) [inline css/types/image.json]: I'm not as familiar with CSS content structure as I with APIs. If `at` gets its own page, then the webkit data needs to be copied.

File: css/types/image.json [hunk 3]
```diff
@@ -405,6 +405,533 @@
                 }
               }
             }
+          },
+          "radial-gradient": {
+            "__compat": {
+              "description": "<code>radial-gradient()</code>",
+              "support": {
+                "webview_android": {
+                  "version_added": true
```
Comment (jpmedley) [inline css/types/image.json]: `webview_android` and `chrome_android` also support the `-webkit` prefix.

File: css/types/image.json [hunk 4]
```diff
@@ -405,6 +405,533 @@
...
+                "webview_android": {
+                  "version_added": "4.4"
+                },
+                "chrome": [
+                  {
+                    "version_added": "26"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": "10"
```
Comment (jpmedley) [inline css/types/image.json]: See my notes above about `-webkit-`.

**Comment by teoli2003** [comment]
Now that PR #2846  is merged, I will schedule some time next week to review this.

File: css/types/image.json [hunk 5]
```diff
@@ -405,6 +406,559 @@
...
+                    "version_added": true
+                  }
+                ],
+                "chrome_android": [
+                  {
+                    "version_added": "26"
+                  },
+                  {
+                    "prefix": "-webkit-",
+                    "version_added": true
```
Comment (ExE-Boss) [inline css/types/image.json]: This should probably be `"18"`.

File: css/types/image.json [hunk 6]
```diff
@@ -466,4 +1020,4 @@
       }
     }
   }
-}
+}
```
Comment (ExE-Boss) [inline css/types/image.json]: Please don’t remove the last newline.

File: css/types/image.json [hunk 7]
```diff
@@ -405,6 +406,559 @@
...
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "12",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
+                    "version_added": "15"
```
Comment (ExE-Boss) [inline css/types/image.json]: The first Opera for Android release which used Chromium was 14, not 15, which was the first desktop Opera release which used Chromium.

See #1712

File: css/types/image.json [hunk 8]
```diff
@@ -405,6 +406,559 @@
...
+                "ie": {
+                  "version_added": "10",
+                  "notes": "Internet Explorer 5.5 through 9.0 supported gradients via a proprietary filter: <code><a href='https://developer.mozilla.org/docs/Web/CSS/-ms-filter#Gradient'>-ms-filter: progid:DXImageTransform.Microsoft.Gradient()</a></code>."
+                },
+                "opera": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
```
Comment (ExE-Boss) [inline css/types/image.json]: This should be `-o-`

File: css/types/image.json [hunk 9]
```diff
@@ -405,6 +406,559 @@
...
+                    "prefix": "-webkit",
+                    "version_added": "15"
+                  }
+                ],
+                "opera_android": [
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
```
Comment (ExE-Boss) [inline css/types/image.json]: ditto

File: css/types/image.json [hunk 10]
```diff
@@ -405,6 +406,559 @@
...
+                    "version_added": "11.6",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
```
Comment (ExE-Boss) [inline css/types/image.json]: `-webkit-`

File: css/types/image.json [hunk 11]
```diff
@@ -405,6 +406,559 @@
...
+                  {
+                    "version_added": "12.1"
+                  },
+                  {
+                    "prefix": "-o",
+                    "version_added": "12",
+                    "version_removed": "15"
+                  },
+                  {
+                    "prefix": "-webkit",
```
Comment (ExE-Boss) [inline css/types/image.json]: ditto

**Comment by Elchi3** [comment]
Needs a rebase now, sorry :)

**Comment by vinyldarkscratch** [comment]
Not a problem!  The merge conflict was 100% my own fault, haha -- I sorted browsers for css/* and then sorted them again for the file this PR affects.  XD  Merge conflicts eliminated now!

**Pull request merged**